### PR TITLE
Merge turret muter, add lamp clamps - By Brandon Sturgeon

### PR DESCRIPTION
**Pull request by brandonsturgeon**

### CODE REVIEW

File: lua/tool_balance/tools/base/lamp.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    LAMP.SetLightFov = clampMethod( LAMP.SetLightFov, {values.fov} )
+    LAMP.SetBrightness = clampMethod( LAMP.SetBrightness, {values.brightness} )
+    LAMP.SetDistance = clampMethod( LAMP.SetDistance, {values.distance} )
+
+    print( "[CFC_Tool_Balance] base/lamp loaded" )
+end
+
+local function waitingFor()
+    local ent = scripted_ents.GetStored( "gmod_lamp" )
+    return ent and ent.t.SetDamage ~= nil
```
Comment (plally) [inline lua/tool_balance/tools/base/lamp.lua:27]: theres no SetDamage for a lamp

Comment (samuelWilliams99) [reply lua/tool_balance/tools/base/lamp.lua:27]: Yeeeeaaahhhh he's right!

Comment (samuelWilliams99) [reply lua/tool_balance/tools/base/lamp.lua:27]: Hey who wrote this??

**Comment by samuelWilliams99** [comment]
Hey nice PR @brandonsturgeon

**Comment by legokidlogan** [comment]
Everything seems to be working fine, but the min values for lamp FOV and distance don't actually clamp. Fortunately, it doesn't appear to allow for any exploitable tricks.

**Pull request merged**